pkgs/tools/networking/netsniff-ng/default.nix

```diff
@@ -20,6 +20,9 @@ stdenv.mkDerivation rec {
     libnetfilter_conntrack libpcap libsodium liburcu ncurses perl
     pkgconfig zlib ];
 
+  # trafgen can call out to cpp to process config files.
+  propagatedBuildInputs = [ stdenv.cc ];
```

Comment:
Just putting `stdenv.cc` in `propagatedBuildInputs` won't make it available at runtime. You need to either patch the source to use nix-store path or use `wrapProgram`.